### PR TITLE
Refactor GeoJSON servicer

### PR DESCRIPTION
**Pull request by aapeliv**

### CODE REVIEW

File: app/backend/src/couchers/servicers/gis.py [hunk 1]
```diff
@@ -2,70 +2,61 @@
...
-            )
-
-            data = json.dumps(out.scalar()).encode("ascii")
+            latest_pages = session.query(func.max(PageVersion.id)).group_by(PageVersion.page_id).subquery()
 
-            return httpbody_pb2.HttpBody(
-                content_type="application/json",
-                data=data,
+            query = (
+                session.query(PageVersion.page_id.label("id"), PageVersion.geom)
```
Comment (kalvdans) [inline app/backend/src/couchers/servicers/gis.py:57]: Does it matter what we label `page_id` in the subquery? I was trying to drop it but don't know which tests to rerun to see if it breaks...

Comment (aapeliv) [reply app/backend/src/couchers/servicers/gis.py:57]: This is very poorly tested currently. But it is used (the `id` goes into a GeoJSON "property") which is used when it's clicked on the map:

https://github.com/Couchers-org/couchers/blob/043ab8bb278f3d733fc7d7cc2fd00be1e60df484/app/frontend/src/features/map/MapPage.tsx#L28-L31

Comment (kalvdans) [reply app/backend/src/couchers/servicers/gis.py:57]: That's fine :)

**Review by kalvdans — APPROVED** [review]

**Comment by aapeliv** [comment]
Thanks. I'll merge this into `communities` whenever the pages stuff has been merged.

**Pull request merged**